modules/developer_manual/pages/core/index.adoc

```diff
@@ -1,3 +1,5 @@
 = Core Development
 
 In this section you will find all the details you need to develop ownCloud’s core.
+
+You should develop starting with the latest code in the core master branch. The core development documentation is written assuming that you are doing that.
```

Comment:
```suggestion
IMPORTANT: You should develop starting with the latest code in the core master branch. The core development documentation is written assuming that you are doing that.
```
Not a must highlighting with important, but creates good visibility. Reject if not ok

Reply:
IMO this is just a note/recommendation/information aimed at developers. So not really necessary to write "IMPORTANT".